generated/nidaqmx/stubs/nidaqmx_pb2.py

```diff
@@ -0,0 +1,1978 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please update CONTRIBUTING.md to say what command line to run to regenerate the gRPC stubs. 

See the note about copying measurementlink-python's generate_grpc_stubs.py before you do this.